src/windows/wsladiag/main.cpp

```diff
@@ -0,0 +1,150 @@
+/*++
+
+Copyright (c) Microsoft. All rights reserved.
+
+Module Name:
+
+    main.cpp
+
+Abstract:
+
+    Entry point for the wsladiag tool, performs WSL runtime initialization and parses --list/--help.
+
+--*/
+
+#include "precomp.h"
+#include "CommandLine.h"
+#include "wslutil.h"
+#include "wslaservice.h"
+#include "WslSecurity.h"
+
+using namespace wsl::shared;
+namespace wslutil = wsl::windows::common::wslutil;
+
+int wsladiag_main(std::wstring_view commandLine)
+{
+    wslutil::ConfigureCrt();
+    wslutil::InitializeWil();
+
+    WslTraceLoggingInitialize(LxssTelemetryProvider, !wsl::shared::OfficialBuild);
```

Comment:
@OneBlue this seems like a strange provider to use here. Is there a more appropriate one?

Reply:
I agree, I think we can just use a single provider for all of WSLA.